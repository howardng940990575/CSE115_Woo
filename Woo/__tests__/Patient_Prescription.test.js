import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Patient_PrescriptionScreen from '../src/Patient_Prescription';

test('Patient Prescription SnapShot', ()=> {
    const snap = renderer.create(<Patient_PrescriptionScreen />).toJSON();
    expect(snap).toMatchSnapshot();
});
