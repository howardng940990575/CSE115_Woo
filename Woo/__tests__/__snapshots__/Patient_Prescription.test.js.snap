// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Patient Prescription SnapShot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#72C3C9",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
     Prescription Screen Homepage
  </Text>
  <View
    accessibilityRole="button"
    accessibilityStates={Array []}
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Sign Out
      </Text>
    </View>
  </View>
</View>
`;
